### PR TITLE
DSEAT-34 Added functionality to remove users after being added in spl…

### PR DESCRIPTION
**Pull request by IremA99**
https://re-motion.atlassian.net/browse/DSEAT-36

### CODE REVIEW

**Review by KuJo7 — APPROVED** [review]

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 1]
```diff
@@ -19,12 +23,25 @@ function AddPerson() {
...
     amountElement.type = "number";
     amountElement.step = ".01";
-    personSpan.append(amountElement);
-    personSpan.append(document.createElement("br"));
+    personDiv.append(amountElement);
+    personDiv.append(document.createElement("br"));
+
+    let button = document.createElement("button");
+    button.innerText = "Remove";
+    button.type = "button";
```
Comment (FelixSchoberRubi) [inline RubeesEat/wwwroot/js/SplitBill.js:31]: button.type is unnecessary, it already creates a button element

**Review by FelixSchoberRubi — APPROVED** [review]

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 2]
```diff
@@ -19,12 +23,24 @@ function AddPerson() {
...
+    button.onclick = function () {
+        personDiv.remove();
+        let addToOptions = document.createElement("option");
+        addToOptions.value = selectedPerson;
+        addToOptions.dataset.id = selectedId;
+        addToOptions.textContent = selectedPerson;
+        selectElement.appendChild(addToOptions);
+    }
+    personDiv.appendChild(button);
+    document.querySelector("#addedPeople").appendChild(personDiv);
```
Comment (a-ctor) [inline RubeesEat/wwwroot/js/SplitBill.js:40]:
```suggestion
    document.getElementById("addedPeople").appendChild(personDiv);
```

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 3]
```diff
@@ -19,12 +23,24 @@ function AddPerson() {
     amountElement.name = "amount" + counter;
     amountElement.type = "number";
     amountElement.step = ".01";
-    personSpan.append(amountElement);
-    personSpan.append(document.createElement("br"));
+    personDiv.append(amountElement);
+    personDiv.append(document.createElement("br"));
+
+    let button = document.createElement("button");
```
Comment (a-ctor) [inline RubeesEat/wwwroot/js/SplitBill.js:29]:
```suggestion
    const button = document.createElement("button");
```

File: RubeesEat/wwwroot/js/SplitBill.js [hunk 4]
```diff
@@ -5,9 +5,13 @@ function AddPerson() {
...
-    const personSpan = document.createElement("span");
-    personSpan.textContent = selectedPerson;
-    document.querySelector("#addedPeople").appendChild(personSpan);
+    const personDiv = document.createElement('div');
+    const textDiv = document.createElement('div');
+
+    personDiv.className = 'personDiv';
+    textDiv.className = 'text';
+    textDiv.textContent = selectedPerson + " ";
+    personDiv.appendChild(textDiv);
```
Comment (a-ctor) [inline RubeesEat/wwwroot/js/SplitBill.js:14]: I would suggest reordering the statements to fit with the variable declarations.

```suggestion
    const personDiv = document.createElement('div');
    personDiv.className = 'personDiv';

    const textDiv = document.createElement('div');
    textDiv.className = 'text';
    textDiv.textContent = selectedPerson + " ";
    personDiv.appendChild(textDiv);
```

**Review by a-ctor — APPROVED** [review]

**Review by FelixSchoberRubi — COMMENTED** [review]
Selenium tests for removing people is missing

**Pull request merged**